### ISSUE TITLE
Broken demo on Binder

### ISSUE THREAD

**Issue by jtpio** [jupyterlab/jupyterlab-demo#145]
## Description

The demo appears to be broken when trying it on Binder:

![image](https://github.com/user-attachments/assets/45e483bc-f992-49cb-999f-f7102974286d)

## Reproduce

https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/HEAD?urlpath=lab/tree/demo

## Expected behavior

JupyterLab should start on Binder.

## Context

Also reported in https://github.com/jupyterlab/jupyterlab/issues/16344

**Comment by jtpio** [comment]
Yes, closing as fixed by https://github.com/jupyterlab/jupyterlab-demo/pull/148!

**Comment by rootsmusic** [comment]
@jtpio This repo is for https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/master?urlpath=lab .  Which repo is for the official demo?  (When I click on the JupyterLab icon in [Project Jupyter's site](https://jupyter.org/try), JupyterLite opens intro.ipynb.  Then the "JupyterLab" heading in intro.ipynb says: "See Also: For a more in-depth tour of JupyterLab with a full environment that runs in the cloud, see [the JupyterLab introduction on Binder](https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/HEAD?urlpath=lab/tree/demo)."  However, that Binder is different from this repo's.)

**Comment by krassowski** [comment]
These are the same repositories though?
- a) https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/master?urlpath=lab
- b) https://mybinder.org/v2/gh/jupyterlab/jupyterlab-demo/HEAD?urlpath=lab/tree/demo

b) just opens the `demo` directory

**Comment by rootsmusic** [comment]
> b) just opens the `demo` directory

@krassowski I don't see a `demo` directory in this repo.

**Comment by krassowski** [comment]
Because it is setup programatically from

https://github.com/jupyterlab/jupyterlab-demo/blob/c058aead3b24d96133b55721faf2d28b4823b9fa/talks.yml#L33-L50

**Comment by rootsmusic** [comment]
> These are the same repositories though?

Yes @krassowski, this is the correct repo.  (I misunderstood you.)  Thanks.
